optax/_src/loss.py

```diff
@@ -358,3 +358,33 @@ def loop_body(prev, x):
   per_seq_loss = -jnp.einsum('bn,bn->b', logalpha_phi_last, one_hot)
 
   return per_seq_loss
+
+
+def kl_div_loss(predictions: chex.Array,
```

Comment:
These are now log-predictions aren't they? If my understanding of this is correct we should reflect this in the name and call it `log_predictions`. That will also make it easier to make this clear in the doc string.

Update: I know this would cause confusion with `log_targets` but I think we should remove the flag (see next comment).

Reply:
Done!